guides/v2.3/graphql/reference/quote.md

```diff
@@ -9,6 +9,9 @@ A quote represents the contents of a customer's shopping cart. It is responsible
 * Determining estimated shipping costs
 * Calculating subtotals, computing additional costs, applying coupons, and determining the payment method
 
+{:.bs-callout .bs-callout-tip}
+Except for `createEmptyCart`, the mutations defined in this topic are not supported in Magento 2.3.1, but will be supported in a future release.
```

Comment:
Should this be 2.3.1 and earlier?

Reply:
good catch!

Reply:
I think it should be mentioned that the mutations are available in the 2.3-develop branch of the graphql-ce repository

Reply:
I know this has merged, but I'd like to make a statement. **NEVER** make promises... "will be supported in future" 
tsk tsk

Reply:
@hguthrie This statement was rewritten in #3919, and there is no longer a promise. However, I wouldn't say NEVER make a promise. This is just one of those odd corner cases where Community contributors need to know about code that was removed from 2.3.1 after code freeze.